src/Raven.Server/Documents/Handlers/BatchRequestParser.cs

```diff
@@ -330,6 +335,46 @@ private async Task<CommandData> MoveNextUnlikely(JsonOperationContext ctx, Blitt
 
                 return await ReadSingleCommand(ctx, _stream, _state, _parser, _buffer, modifier, _token);
             }
+
+            public async Task<byte[]> GetRavenData(long dataSize)
```

Comment:
We can't handle this. We need to write it to a temporary file, like we do with normal attachment load.
We can avoid that if we are using small files, but otherwise, need a stream